src/lib/base/component-manager.js

```diff
@@ -64,6 +64,27 @@ const nonBubblingEventsMap = {
 // match & select "[eventType] [css selector]"
 const handlerMethodPattern = new RegExp(`^(${events.join('|')}|${Object.keys(nonBubblingEventsMap).join('|')}) (.*)`);
 
+/**
+ * Injects fallback mouse events for non-touch devices to events object.
+ * 
+ * @param {!Object} events
+ */
+function injectFallbackMouseEvents(events) {
+    const fallbackableEvents = ['touchend', 'tap'];
+
+    const _events = { ...events };
+
+    fallbackableEvents.forEach(eventType => {
+        if (events['click'] || !events[eventType]) {
+            return;
+        }
+
+        _events['click'] = _events[eventType];
```

Comment:
what if `touchend` and `tap` are different handlers? in this case the last handler would win, right? is it possible to do this without a loop every time we create a new component? as in `events['touchend'] || events['tap']`?

Reply:
That would make more sense, will do it 👍